src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy

```diff
@@ -147,11 +153,13 @@ class HockeyAppUploadTask extends DefaultTask {
             }
         }
 
-        uploadAppplicationFileToHockeyApp(applicationFile, mappingFile, appId)
+        applicationFiles.each {
+            uploadApplicationFileToHockeyApp(it, mappingFile, appId)
```

Comment:
I have tried to upload multiple APKs to HockeyApp and looks like it's ok. All crash reports are collected in the one of them (probably the last one) and you can download each of them separately. Don't know, what HockeyApp guys think about it.